src/Parser/Core/Modules/StatTracker.js

```diff
@@ -236,13 +236,17 @@ class StatTracker extends Analyzer {
     \****************************************/
 
     // region Azerite Traits
+    [SPELLS.GEMHIDE.id]: GEMHIDE_STATS,
+    [SPELLS.METICULOUS_SCHEMING.id]: combatant => Object.values(combatant.traitRanks(SPELLS.METICULOUS_SCHEMING.id)).reduce((haste, rank) => {
+      return haste + calculateAzeriteEffects(SPELLS.METICULOUS_SCHEMING.id, rank)[0];
+    }, 0),
```

Comment:
Gemhide uses a helper defined inside the Gemhide module, but because I use `statTracker` as dependancy in my Module that causes a circular import.

I think if I clean it up with an easy to use helper it would be fine to define it in here...